[Chat][Document] Update README for chat demo App

lpx1989 opened the pull request:
## Purpose
update readme for chat demo App.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Documentation content changes
- [ ] Other... Please describe:

## Pull Request Checklist


- [ ] Public API changes
- [ ] Verified for cross-platform
- [ ] Synced with iOS, Web team
- [ ] Internal review completed
- [ ] UI Changes
  - [ ] Screen captures included
  - [ ] Tested for Light/Dark mode
  - [ ] Tested for screen rotation
  - [ ] Tested on Tablet and small screen device (5")
  - [ ] Include localization changes
- [ ] Tests
  - [ ] Memory leak analysis performed
  - [ ] Tested on API 21, 26 and latest 
  - [ ] Tested for background mode
  - [ ] Unit Tests Included
  - [ ] UI Automated Tests Included

iaulakh reviewed (changes requested):
Peixin, would it be possible to add the variant selection for demo app. Now developers also need to choose variant either call or chat.

iaulakh commented on azure-communication-ui/demo-app/README.md:
```diff
@@ -30,4 +30,9 @@ library. Showcases use of both Java and Kotlin to run library.
+    - `IDENTITY`="..."            # the identity for chat
+    - `CREDENTIAL`="..."          # the credential for chat
+    - `THREAD_ID`="..."           # chat thread id
+    - `SENDER_DISPLAY_NAME`="..." # the display name for local participant in chat thread
```
we have not added below:
CREDENTIAL -> ACS_TOKEN will be used
SENDER_DISPLAY_NAME -> USER_NAME will be used

lpx1989 commented:
> Peixin, would it be possible to add the variant selection for demo app. Now developers also need to choose variant either call or chat.

addressed and updated
